### PR TITLE
UG References to "sign-in" are inconsistent

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation ticket: https://mattermost.atlassian.net/browse/MM-33322

Updated:
- User's Guide > Getting Started > Signing In > Sign In
   - Updated "sign-in page" references to "sign-in screen"

### CODE REVIEW

**Comment by cwarnermm** [comment]
@justinegeffen - We have 2 instances of numbered sub-steps that don't display as intended:
- https://docs.mattermost.com/help/getting-started/signing-in.html#ios-setup
- https://docs.mattermost.com/help/getting-started/signing-in.html#android-setup

I tried a number of things to get the a. and b. steps to display but wasn't successful. Do you know the way to correct these?

File: source/help/getting-started/signing-in.rst [hunk 1]
```diff
@@ -121,25 +121,24 @@ iOS Setup
 
 Your Mattermost teams can be accessed on iOS mobile devices via the Mattermost Mobile App.
 
-#. Open the `App Store` on your Apple device running iOS 9.0 or later.
-#. Search for “Mattermost” and select **GET** to download the app.
-#. Open Mattermost from your homescreen and enter your team and account information to log in:
+1. Open the `App Store` on your Apple device running iOS 9.0 or later.
```
Comment (justinegeffen) [inline source/help/getting-started/signing-in.rst:124]:
```suggestion
1. Open the App Store on your Apple device running iOS 9.0 or later.
```

File: source/help/getting-started/signing-in.rst [hunk 2]
```diff
@@ -121,25 +121,24 @@ iOS Setup
...
+2. Search for “Mattermost” and select **GET** to download the app.
```
Comment (justinegeffen) [inline source/help/getting-started/signing-in.rst:125]:
```suggestion
2. Search for "Mattermost" and select **GET** to download the app.
```

File: source/help/getting-started/signing-in.rst [hunk 3]
```diff
@@ -121,25 +121,24 @@ iOS Setup
...
+3. Open Mattermost from your homescreen and enter your team and account information to log in:
 
-   #. **Enter Server URL:** This is the web address you go to when you want to access Mattermost. You can find the Server URL by asking your
-      System Admin or by looking at the address bar in a desktop browser tab with Mattermost open. It is in the format ``https://domain.com``.
-   #. **Sign in to Mattermost:** This is your account login information as described by one of the sign in methods above.
+   a. **Enter Server URL:** This is the web address you go to when you want to access Mattermost. It is in the format ``https://domain.com``. You can find the Server URL by asking your System Admin, or by looking at the address bar in a desktop browser tab with Mattermost open.
```
Comment (justinegeffen) [inline source/help/getting-started/signing-in.rst:128]:
```suggestion
   a. **Enter Server URL:** This is the web address you go to when you want to access Mattermost. It's in the format ``https://domain.com``. You can find the Server URL by asking your System Admin, or by looking at the address bar in a desktop browser tab with Mattermost open. 
```

File: source/help/getting-started/signing-in.rst [hunk 4]
```diff
@@ -121,25 +121,24 @@ iOS Setup
...
 
 Android Setup
 -------------
 
 Your Mattermost teams can be accessed on Android mobile devices by downloading the Mattermost Mobile App.
 
-#. Open the `Google Play Store`_ on your Android device.
-#. Search for “Mattermost” and select **INSTALL** to download the app.
-#. Open Mattermost from your homescreen and enter your team and account information to login:
+1. Open the `Google Play Store`_ on your Android device.
```
Comment (justinegeffen) [inline source/help/getting-started/signing-in.rst:136]:
```suggestion
1. Open the Play Store on your Android device.
```

File: source/help/getting-started/signing-in.rst [hunk 5]
```diff
@@ -121,25 +121,24 @@ iOS Setup
...
+2. Search for “Mattermost” and select **INSTALL** to download the app.
```
Comment (justinegeffen) [inline source/help/getting-started/signing-in.rst:137]:
```suggestion
2. Search for "Mattermost" and select **INSTALL** to download the app.
```

File: source/help/getting-started/signing-in.rst [hunk 6]
```diff
@@ -121,25 +121,24 @@ iOS Setup
...
+3. Open Mattermost from your homescreen and enter your team and account information to login:
 
-   #. **Enter Server URL:** This is the web address you go to when you want to access Mattermost. You can find the Server URL by asking your System Admin or by looking at the address bar in a desktop browser tab with Mattermost open. It is in the format ``https://domain.com``.
-   #. **Sign in to Mattermost:** This is your account login information as described by one of the sign in methods above.
+   a. **Enter Server URL:** This is the web address you go to when you want to access Mattermost. It is in the format ``https://domain.com``. You can find the Server URL by asking your System Admin, or by looking at the address bar in a desktop browser tab with Mattermost open.
```
Comment (justinegeffen) [inline source/help/getting-started/signing-in.rst:140]:
```suggestion
   a. **Enter Server URL:** This is the web address you go to when you want to access Mattermost. It's in the format ``https://domain.com``. You can find the Server URL by asking your System Admin, or by looking at the address bar in a desktop browser tab with Mattermost open. 
```

**Review by justinegeffen — APPROVED** [review]
Some minor (non-blocking) suggestions but otherwise LGTM.

**Pull request merged**